scale.yml

```diff
@@ -13,6 +13,19 @@
   vars:
     ansible_connection: local
 
+- hosts: localhost
+  tasks:
+    - name: deploy warning for non kubeadm
+      debug:
+        msg: "non-kubeadm code deprecated in v2.9  (TODO remove in v3.0)"
+      when: not kubeadm_enabled and not skip_kubeadm_warning 
+
+    - name: deploy cluster for non kubeadm
+      pause:
+        prompt: "Are you sure you want to deploy cluster by non-kubeadm. If you want to do that, Please press the 'Enter' button!!!"
```

Comment:
i think word button is not needed here :)

Reply:
💯